static/src/javascripts/__flow__/types/ab-tests.js

```diff
@@ -70,6 +71,7 @@ declare type InitEpicABTest = {
     locations?: string[],
     locationCheck?: () => boolean,
     dataLinkNames?: string,
+    campaignCode?: string,
```

Comment:
Don't think you need this here if you got it in the definition of `AcquisitionsABTest` above